apps/system/js/app_window_manager.js

```diff
@@ -527,15 +527,17 @@
         case 'appcreated':
           var app = evt.detail;
           this._apps[evt.detail.instanceID] = app;
+          Service.request('registerAudioChannel', app);
```

Comment:
I didn't remember if it's me telling you to put this here...
Do that here will not cover: lockscreen/secure window/attention window cases.
Choose one solution:
1. register audio channel in each window's constructor and destructor
2. listen to *created/*terminated event in audioChannelService